### ISSUE TITLE
Further support

### ISSUE THREAD

**Issue by Lethality5001** [ThirteenAG/GTAIV.EFLC.FusionFix#224]
Will it get backported correctly on patch 1.0.7.0 and 1.0.8.0? Now that the "beef" is over, lets make GTAIV great?

**Comment by cpmodding** [comment]
Have you tried to run FF on such patches?

**Comment by Lethality5001** [comment]
> Have you tried to run FF on such patches?

yes i did, and seems to work 95% of it, only issue seems to be Motion blur and bloom get put "on" or "off" at the same time and shadow filter options are a bit glitchy in the graphics menu but so far works well, he should make a port

**Comment by LLIXXXIR** [comment]
2.0 update very good, but in backward capability i have no work RecoilFix on 1.0.7.0, while in ini file this option on. And new version not start, writes something about live account after installing mod. Stop, mod make me buy game? GTA modders make me buy TakeTwo game? It should be mistake..

**Comment by Meesmoth** [comment]
Version 2.0 should have support for patch 7 & 8; it was previously released as beta since September, but as far as I can see they got rid of it in the final release. Maybe by 3.0 then, but don't expect anything as even other forks for patch 7 & 8 were also taken down, as if we're being forced to switch to CE.
